pkg/codegen/testing/test/program_driver.go

```diff
@@ -109,6 +109,10 @@ var PulumiPulumiProgramTests = []ProgramTest{
 		Directory:   "aws-iam-policy",
 		Description: "AWS IAM Policy",
 	},
+	{
+		Directory:   "read-file-func",
```

Comment:
We already had the test data, but it wasn't hooked up in a test?

Reply:
I added this test a year ago but somehow it was removed from this list 😅 I asked on Slack if someone knows why it was removed but I am putting back in anyways

Maybe we should check for directories that are in testdata but aren't part of the SDK tests 🤔

Reply:
To close the loop publicly, I believe this was a mismerge and we accidentally deleted this here: https://github.com/pulumi/pulumi/pull/10958/files#diff-860749d2dad497a2e4dbdb2e89ceb7ebc99bed8add0ba1ec6a7009946085d787L90